iina/Logger.swift

```diff
@@ -45,31 +56,97 @@ struct Logger {
   static let enabled = Preference.bool(for: .enableAdvancedSettings) && Preference.bool(for: .enableLogging)
 
   static let logDirectory: URL = {
+    // get path
+    let libraryPath = FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask)
+    guard libraryPath.count >= 1 else {
+      fatalDuringInit("Cannot get path to Logs directory: \(libraryPath)")
+    }
+    let logsUrl = libraryPath.first!.appendingPathComponent("Logs", isDirectory: true)
```

Comment:
```suggestion
    let libraryPaths = FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask)
    guard libraryPath = libraryPaths.first else {
      fatalDuringInit("Cannot get path to Logs directory: \(libraryPath)")
    }
    let logsUrl = libraryPath.appendingPathComponent("Logs", isDirectory: true)
```

Reply:
This was code I moved out of Utility and didn't cleanup. Fixed.